src/core/gridSystem.js

```diff
@@ -23,7 +23,7 @@ module.exports = function addGrid(map, onChangeLocation) {
     include: L.Mixin.Events,
 
     options: {
-      cellSize: 64,
+      cellSize: 40,
```

Comment:
OK, so here you'd want to either use something like `options.cellSize` and have it settable via the constructor (like `options.map`, above), OR you could make addGrid have a method called `addGrid.setCellSize(degreesWide, degreesTall)`